src/Http/UrlBuilder.php

```diff
@@ -6,36 +6,24 @@
 
 class UrlBuilder
 {
-    /** @var string */
-    protected $url;
+    protected string $url;
```

Comment:
This is also null by default in the original code. The constructor code can skip setting it, therefore we have to have a default value (implied `null` from the previous version)
```suggestion
    protected ?string $url = null;
```

Reply:
Good catch, I will make it nullable! (it's actually not used at all, `getUrl()` builds the URL using all the other bits).